### DISCUSSION TITLE
Upgrading to turbo 1.8.5 fails in vercel build

### DISCUSSION THREAD

**Discussion by MarcusHSmith** [vercel/community#1851] [Help]
### Summary

`prim-ui:build: Error: The package "esbuild-linux-64" could not be found, and is needed by esbuild.`

It build successfully locally but fails in Vercel

![Uploading Screenshot 2023-03-22 at 07.09.58.jpg…]()

**Comment by nathanhammond** [comment]
Are you using npm? If so you can have all sorts of problems with this:

https://github.com/npm/cli/issues/4828
